README.md

```diff
@@ -163,6 +163,7 @@ Community Maintained Cars and Features
 | Subaru    | Crosstrek 2018-19             | EyeSight          | Stock            | 0mph               | 0mph         |
 | Subaru    | Forester 2019                 | EyeSight          | Stock            | 0mph               | 0mph         |
 | Subaru    | Impreza 2017-19               | EyeSight          | Stock            | 0mph               | 0mph         |
+| Subaru    | Legacy 2015-18                | EyeSight          | Stock            | 0mph               | 0mph         |
```

Comment:
Revert the README change, we'll add it once it's not dashcam only

Reply:
done